ofono/include/sms.h

```diff
@@ -60,6 +61,12 @@ struct ofono_sms_driver {
 				ofono_sms_bearer_set_cb_t, void *data);
 };
 
+enum ofono_sms_data_flag {
+	OFONO_SMS_DATA_FLAG_DELIVERY_REPORT =		0x01,
+	OFONO_SMS_DATA_FLAG_USE_BIG_ENDIAN =		0x02,
+	OFONO_SMS_DATA_FLAG_USE_LITTLE_ENDIAN =		0x04,
```

Comment:
It's a bit funny to be able to ask for both little and big endian, but having a "troolean" would take the same 2 bits, so I guess not much point in doing anything else.

Reply:
Yeah we could manage only with little endian flag here for sure. Maybe we should drop that? It is not anyways used and as this is includable header the value change does not matter. LE is after all an exception to the norm.